doc/whatsnew/fragments/6871.user_action

```diff
@@ -0,0 +1,14 @@
+* The compare to empty string checker (``pylint.extensions.emptystring``) and the compare to
+  zero checker (``pylint.extensions.compare-to-zero``) have been removed and their checks are
+  now part of the implicit booleaness checker:
+
+  - ``compare-to-zero`` was renamed ``use-implicit-booleaness-not-comparison-to-zero``
+    and ``compare-to-empty-string`` was renamed ``use-implicit-booleaness-not-comparison-to-string``
+    and they now need to be enabled explicitly.
+  - The `pylint.extensions.emptystring`` and ``pylint.extensions.compare-to-zero`` extensions no longer exists and
+    needs to be removed from the ``load-plugins`` option.
```

Comment:
Really considering to make an update tool for configuration files, it's not a lot of work and will permit to make breaking changes with more serenity.